topics/contributing/tutorials/slides-with-video/tutorial.md

```diff
@@ -75,30 +75,31 @@ The captioning process is completely automated, but it means that for very long
 
 Every sentence must end with punctuation like `.` or `?` or even `!` if you're feeling excited. We merge all lines of the presenter notes before having them spoken by Polly. If you forget the punctuation it will be one long run-on sentence.
 
-### Abbreviations and "Weird" Names
+### Abbreviations
 
-In the captions you will want to spell out abbreviations
+These arae generally fine as-is. Make sure they are all caps though.
 
-> > **Good**
-> > This role deploys C V M F S.
-> {: .code-in}
->
-> > **Bad**
-> > This role deploys CVMFS.
-> {: .code-out}
-{: .code-2col}
+> **Good**
+> This role deploys CVMFS.
+{: .code-in}
 
-For many terms we read them differently than they're written, e.g. 'src' vs 'source'. Most of us would pronounce it like the latter, even though it isn't spelt that way. Our speaking robot doesn't know what we mean, so we need to spell it out properly.
+### "Weird" Names
 
-> > **Good**
-> > Copy copies a file from src on localhost...
-> {: .code-in}
->
-> > **Bad**
-> > Copy copies a file from source on localhost...
-> {: .code-out}
-{: .code-2col}
+In the captions you will want to teach the GTN how to pronounce these words by editing `bin/ari-map.yml` to provide your definition.
+
+E.g.
+
+Word       | Pronunciation
+---------- | ---
+SQLAlchemy | SQL alchemy
+FastQC     | fast QC
+nginx      | engine X
+gxadmin    | GX admin
+/etc       | / E T C
+
+This is the same for the many terms we read them differently than they're written, e.g. 'src' vs 'source'. Most of us would pronounce it like the latter, even though it isn't spelt that way. Our speaking robot doesn't know what we mean, so we need to spell it out properly.
```

Comment:
That's great! If we could only have a way to tell it how to discriminate (in context) between the 2 pronunciations of "analyses": the plural of "analysis" vs. the 3rd person singular present tense of "to analyse": https://www.youtube.com/watch?v=o2LRIgxjeP0

Reply:
So in *theory* that should be taken care of by context with the TTS model and not our responsibility! Which is super nice. But there seem to be some edge cases with AWS. E.g. "Many analyses were done" is pronounced correctly, but "together we did multiple analyses" doesn't. So, context matters for the engines, and we may need to rephrase things occasionally for this.

Reply:
Ah it's wrong I guess in the intro slides then. Ok, I'll see if I can find an OK rephrasing, thanks for alerting on that!